core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -623,9 +623,12 @@ class BrokerServer(
     }
   }
 
-  override def shutdown(): Unit = {
+  override def shutdown(): Unit = shutdown(TimeUnit.MINUTES.toMillis(5))
```

Comment:
How about adding default implementation to parent class?

Reply:
Yup, that makes sense. Changed